src/rantanplan/core.py

```diff
@@ -99,69 +99,96 @@ def have_prosodic_liaison(first_syllable, second_syllable):
             and second_syllable['syllable'][0] in LIAISON_SECOND_PART)
 
 
-def get_synalephas(words):
+def get_synalephas(sinaereris_words_dictionary):
+    """
+    Gets a list of dictionaries for each word on each line of the poem
+    and joins the syllables to create phonologic syllables according to its
+    'has_synaloepha' value
+    :param sinaereris_words_dictionary: List of dictionaries for each word of the poem
+    :return: A list of conjoined syllables
+    """
     prosodic_line = []
-    word_list = copy.deepcopy(words)
+    word_list = copy.deepcopy(sinaereris_words_dictionary)  # create a deepcopy since we're going to pop()
```

Comment:
Why the change? Why not `words_dict`? Just wondering.

Reply:
To make clear that we are working with sinaeresis, clarity I guess?

Reply:
A bit verbose, IMHO

Reply:
changed it to sinaeresis_words